website/src/js/app.js

```diff
@@ -121,4 +121,42 @@ var eventInfoLoad = function() {
   }
 }
 
-eventInfoLoad();
\ No newline at end of file
+eventInfoLoad();
+
+// Widgets section
+function widgetsCloud() {
```

Comment:
I'm curious why you moved this from a separate script called from the widgets page into the main js.

Reply:
My point was to reduce the number of requests made, but I'll revert this back to widgets.js, no problem!